vars/cloverReportPublish.groovy

```diff
@@ -74,8 +74,8 @@ void call(Map config = [:]) {
                 fileOperations([fileRenameOperation(source: 'test.cov',
                                                 destination: new_name)])
           }
-        } catch(Hudson.AbortException ex) {
-            println ("Unstasn failed: ${ex}")
+        } catch (hudson.AbortException ex) {
+            println("Unstasn failed: ${ex}")
```

Comment:
Would be nice to fix this spelling error here while editing this line.
```suggestion
            println("Unstash failed: ${ex}")
```

Reply:
I don't think it needs to wait for a full CI.  And it choked due to a release/2.2 RPM rebuild&upload in any case.

Land when you wish.